src/index.js

```diff
@@ -73,10 +73,17 @@ const eventEmitter = getEventEmitter();
 function addListener(keyCommand, callback) {
     registerKeyCommands([keyCommand]);
     const event = eventEmitter.addListener('onKeyCommand', (response) => {
-        if (response.input !== keyCommand.input || response.modifierFlags !== keyCommand.modifierFlags) {
+        if (!response.input || !response.input.length) {
             return;
         }
 
+        /**
+         * Normalizing input to lowercase, android will return returns keyEvent.getDisplayLabel
+         * which is the label that is physically printed on the keyboard.
+         */
+        if (response.input.toLowerCase() !== keyCommand.input.toLowerCase() || response.modifierFlags !== keyCommand.modifierFlags) {
```

Comment:
Aren't you converting the input to lowercase on line 72 of `android/src/main/java/com/expensify/reactnativekeycommand/KeyCommandModule.java`.
```
 params.putString("input", String.valueOf(keyEvent.getDisplayLabel()).toLowerCase());
```